CASE:
    hardware_interface:
        loop_hz: 10 # hz
        joints:
            # need to be defined in the order that they run in CAN
            - left_front_wheel
            - left_rear_wheel
            - right_front_wheel
            - right_rear_wheel
